src/controller/python/ChipDeviceController-ScriptDevicePairingDelegate.h

```diff
@@ -35,8 +35,8 @@ namespace Controller {
 extern "C" {
 typedef void (*DevicePairingDelegate_OnPairingCompleteFunct)(PyChipError err);
 typedef void (*DevicePairingDelegate_OnCommissioningCompleteFunct)(NodeId nodeId, PyChipError err);
-typedef void (*DevicePairingDelegate_OnWindowOpenCompleteFunct)(NodeId nodeId, uint32_t setupPinCode, const char * setupCode,
-                                                                PyChipError err);
+typedef void (*DevicePairingDelegate_OnWindowOpenCompleteFunct)(NodeId nodeId, uint32_t setupPinCode, const char * manualCode,
+                                                                const char * setupCode, PyChipError err);
```

Comment:
suggest rename to setupQrCode

Reply:
Thank you for your suggestion, it has been revised